### DISCUSSION TITLE
Feature Request: Drag & Drop compatibility in main prompt textbox

### DISCUSSION THREAD

**Discussion by HaikenEdge** [TheLastBen/fast-stable-diffusion#1926] [Ideas]
With the current terrible Gradio performance, I've found that I need to start the final step of the notebook quite often, even though the prompt I want to generate with is mostly the same between generations.

It'd be nice if functionality could be added where I could just select and drag the prompt from one instance of the UI to the next; while the negative prompt text box is receptive to drag-and-drop, the positive prompt text box isn't, and I think it'd greatly enhance the UI's functionality of this functionality was added.

**Comment by TheLastBen** [comment]
if you use cloudflare tunnel + latest_working_commit, there shouldn't be a freezing issue.

And if you want to bring back the previous prompt directly to gradio, click on the blue button with an arrow in it under the generate button and it will load back the previous prompt+settings

**Reply by HaikenEdge** [reply]
cloudflare tunnel + latest_working_commit still seem doesn't stop the Generate button from not working; however, I hadn't thought of the blue button, so thanks for that.

Edit: In fact, taking off the cloudflare tunnel actually stopped the issue with the Generate button not working for me.
